README.md

```diff
@@ -3,15 +3,17 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T1099616)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# Blazor Editors - How to create editors dynamically according to the EditForm's model
 
-* [UserData.cs](./CS/DxBlazorApplication1/Data/UserData.cs)
-* [Index.razor](./CS/DxBlazorApplication1/Pages/Index.razor)
-<!-- default file list end -->
-# Blazor Editors - How to create editors dynamically based on a data object
+This example illustrates how to add the [Form Layout](https://docs.devexpress.com/Blazor/401181/navigation-and-layout#form-layout) component to an [EditForm](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.forms.editform?view=aspnetcore-7.0) at runtime. The Form Layout component is populated with [DevExpress Data Editors](https://docs.devexpress.com/Blazor/401156/data-editors) dynamically according to the EditForm's model.
+
+![image](image.png)
 
-<p>The sample illustrates how to create editors dynamically in EditForm. In the sample, DxFormLayout is created and its layout items with Blazor DevExpress Editors are created dynamically based on data object properties and attributes of these properties.</p>
+## Files to Review
 
+* [UserData.cs](./CS/DxBlazorApplication1/Data/UserData.cs)
+* [Index.razor](./CS/DxBlazorApplication1/Pages/Index.razor)
 
+## Documentation
 
```

Comment:
Is the image below editable? If so, can you change "Register to DevExpress" to "Register with DevExpress"?

Reply:
I've updated the image.

Reply:
Thank you!